modules/ztp-hub-factory-pipeline.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: CONCEPT
+[id="hub-factory-pipeline_{context}"]
+= The hub factory pipeline
+
+The hub configuration pipeline stage prepares the hub cluster to deploy multiple edge clusters for the end customer.
+
+.Prerequisites
```

Comment:
Same as another comment - .Prerequisites heading looks odd here without it being an actual procedure module. Maybe consider removing that or rewording somehow

Reply:
Prerequisites covered elsewhere so duplication. I have removed.